var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -81,7 +84,8 @@ class Crtm(CMakePackage):
 
     def url_for_version(self, version):
         if self.spec.satisfies("@v3") or version >= Version("3.0.0"):
```

Comment:
This check doesn't work as you might expect. First,`self.spec.satisfies()` won't work when this method is called because there are no versions associated with the spec. Even if the value of the `version` argument was available to the method, prepending `v` to the version name isn't recognized for comparison purposes (see https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison). 

I can confirm *all* of the sha256s with the following (or similar) change:

```suggestion
        if version > Version("v3") or version >= Version("3"):
```

though you could leave the second as "`3.0.0`" and it still works.